Actions/Deploy/action.yaml

```diff
@@ -26,11 +26,16 @@ inputs:
   deploymentEnvironmentsJson:
     description: The settings for all Deployment Environments
     required: true
+outputs:
```

Comment:
Maybe also some doc in the action's README?

Reply:
done